daemon/algod/api/algod.oas3.yml

```diff
@@ -5596,6 +5647,20 @@
     "/v2/transactions/simulate": {
       "post": {
         "operationId": "SimulateTransaction",
+        "parameters": [
+          {
+            "description": "Configures whether the response object is JSON or MessagePack encoded.",
```

Comment:
Is there a default? Should it be mentioned?

Reply:
Yes it defaults to JSON. I'll update the description to say that.

Note this will affect other endpoints which use the same definition for this param -- I've checked and all of them default to JSON as well